base/applications/regedit/find.c

```diff
@@ -223,14 +223,28 @@ BOOL RegFindRecurse(
                                   NULL, &cb);
         if (lResult != ERROR_SUCCESS)
             goto err;
-        pb = malloc(cb);
+        pb = malloc(cb + 3); /* To avoid buffer overrun, append 3 NULs */
         if (pb == NULL)
             goto err;
         lResult = RegQueryValueExW(hSubKey, ppszNames[i], NULL, &type,
                                   pb, &cb);
         if (lResult != ERROR_SUCCESS)
             goto err;
 
+        if (type == REG_SZ || type == REG_EXPAND_SZ || type == REG_MULTI_SZ)
```

Comment:
I would either exclude `REG_MULTI_SZ` or support it properly. Right now you'd only be looking at the first string.
(also note that supporting it properly means you need to ensure it's double-null terminated, unlike the other cases)

Reply:
Done. Thanks.